source/config_test.go

```diff
@@ -28,10 +28,11 @@ func TestParse(t *testing.T) {
 		cfg map[string]string
 	}
 	tests := []struct {
-		name    string
-		args    args
-		want    Config
-		wantErr bool
+		name        string
+		args        args
+		want        Config
+		wantErr     bool
+		expectedErr string
```

Comment:
Nitpick: given that we have `expectedErr`, we don't need `wantErr`, because `wantErr` is the same as`expectedErr != nil`.